src/components/ArtifactCard.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+  <p-card class="artifact-card" :class="classes.root">
+    <div class="artifact-card__body" :class="classes.body">
+      <header class="artifact-card__header" :class="classes.header">
+        <h6 class="artifact-card__subheader" :class="classes.subheader">
+          {{ artifact.type }}
+        </h6>
+        <h3 v-if="hasKey">
+          {{ artifact.key }}
+        </h3>
+
+        <template v-if="hasRun && crumbs.length">
+          <p-bread-crumbs :crumbs="crumbs" class="artifact-card__bread-crumbs" :class="classes.breadCrumbs" />
+        </template>
+      </header>
+
+      <div class="artifact-card__summary-container" :class="classes.summaryContainer">
+        <div
+          class="artifact-card__summary-item"
+          :title="artifact.updated.toLocaleString()"
```

Comment:
What's the thought behind these titles attrs? The same info is present on the screen. Won't a screen reader end up reading this date twice?

Reply:
I actually think we should be doing this more but I've removed this instance in favor of something holistic down the line. I don't think this is an issue for screen readers in general (I might be wrong here) but the idea is to provide a locale-compliant browser-supplied timestamp in addition to whatever our display format is. This is particularly helpful in places where we use a relative datetime format (this used to which is what spurred the idea)

Reply:
Ah yes for relative times that would totally make sense 👍